src/components/TaxPayer/Address.tsx

```diff
@@ -10,13 +10,13 @@ interface AddressProps {
   allowForeignCountry?: boolean
 }
 
-export default function AddressFields (props: AddressProps): ReactElement {
+export default function AddressFields(props: AddressProps): ReactElement {
   const {
     checkboxText = 'Check if you have a foreign address',
     allowForeignCountry = true
   } = props
 
-  const { control } = useFormContext<{isForeignCountry: boolean }>()
+  const { control } = useFormContext<{ isForeignCountry: boolean }>()
```

Comment:
surprised our linter isn't catching this.

Reply:
Yeah, it's little things like this that get me. As I mentioned, the line breaks are brutal at first but since it does its thing on save, you eventually don't even think about it which is the point. Without it, you leave it to the developer and you'll get mixed results and you'll be the linter in the PRs. For me it's not a tough call since I've used it for years, but it is an acquired taste.